sdk/cognitivelanguage/ai-language-text/package.json

```diff
@@ -112,7 +112,7 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "cross-env": "^7.0.2",
-    "dotenv": "^8.2.0",
```

Comment:
The recorder offers env that works in both node and browser.
I'd say we don't even need to depend on dotenv.
This PR might be a good point to check out since there aren't too many packages?

Reply:
I believe we need `dotenv` for samples, right?